lib/skate/detours/trip_modification_publisher.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Skate.Detours.TripModificationPublisher do
```

Comment:
Curiosity/Non-blocking: When we add `Shape` publishing as well, do you envision us adding a whole new GenServer like `ShapePublisher`, or would it make more sense to genericize this as `GtfsPublisher`, or `MqttPublisher` or something like that?

Reply:
I don't envision the name changing for that reason, IMO, publishing a shape is also part of Trip Modifications, so I'd imagine it being another parameter to `publish` and a second call to `MqttConnection.publish` within the `handle_call` function.

Reply:
I'm okay with that (especially for now), but what if something about a detour changes other than its `Shape`. For instance, if it goes longer than we originally thought, and so we add more `selected_trips`? Then we probably wouldn't want to re-publish the `Shape`, but just an updated `TripModification` that references the original `Shape`.

That can be a future problem though.